babel.config.js

```diff
@@ -3,7 +3,25 @@
 //   https://babeljs.io/docs/en/config-files
 
 module.exports = {
-  presets: [
-    'module:metro-react-native-babel-preset',
-  ],
+  // Use different presets depending on ${BABEL_ENV:-${NODE_ENV:-development}}.
+  // https://babeljs.io/docs/en/6.26.3/babelrc#env-option
+  env: {
+    // Most of our project is bog-standard React Native. We therefore use the
+    // RN-provided presets by default.
+    development: {
+      presets: ['module:metro-react-native-babel-preset'],
+    },
```

Comment:
... I was completely unaware that `NODE_ENV` was a thing for us. (Who sets that, under our build setup?)

Fortunately this is a `.js` file rather than a `.json` file, so that's an easy fix.

Reply:
I'm not sure how the information is wired through to here. But setting the config here differently for the `env` values of `development` and `production` is how we've previously done different things in debug builds and release builds. Details in the history of this file.